src/app/@ansyn/menu/menu/menu.component.html

```diff
@@ -17,7 +17,7 @@
 						   [hidden]="hideBadge(menuItem.badge)">{{menuItem.badge}}</span>
 					{{menuItem.name | translate}}
 				</div>
-				<div class="menu-btn" (click)="goToLandingPage()">
+				<div class="menu-btn" (click)="onHelp()">
```

Comment:
it not connect to your pr.
but see the comment above.

Reply:
@tltk90 As currently implemented, "help" is part of the menu, and  not a menu item, so the above seems ok